staging/src/k8s.io/api/testdata/v1.24.0/policy.v1beta1.PodSecurityPolicy.json

```diff
@@ -1,149 +0,0 @@
-{
```

Comment:
restore the policy.v1beta1.PodSecurityPolicy files under k8s.io/api/testdata

Reply:
Done.